src/aspire/source/micrograph.py

```diff
@@ -262,8 +272,16 @@ def _images(self, indices):
                 )
             # Assign to array, implicitly performs casting to dtype
             micrographs[i] = micrograph.asnumpy()
+            # Assert pixel_size
+            if (
+                micrograph.pixel_size is not None
+                and micrograph.pixel_size != self.pixel_size
+            ):
+                raise NotImplementedError(
```

Comment:
`ValueError`? (It's not like there's some unimplemented functionality (rescaling?) that we should have here, right?)

Reply:
Sure, changed both occurances.

Reply:
Thanks!